tests/generative/variable_strategies.py

```diff
@@ -408,11 +408,16 @@ def filtered_table(draw):
 
     @st.composite
     def sorted_frame(draw):
-        # select a table which may already have been filtered
-        source = draw(many_rows_per_patient_frame())
-        # Now apply 1-3 sorts
-        for _ in range(draw(st.integers(min_value=1, max_value=3))):
-            source = draw(sort(source))
+        # Decide how many Filters and Sorts we're going to apply
+        num_filters = draw(st.integers(min_value=1, max_value=6))
+        num_sorts = draw(st.integers(min_value=1, max_value=3))
+        # Mix up the order of operations
+        operations = [filter_] * num_filters + [sort] * num_sorts
+        shufffled_operations = draw(st.permutations(operations))
```

Comment:
nit: extra f in shuffled

Reply:
:joy: The perils of autocomplete propagating my typos! That said, I kind of like it: I think it implies that the list is _even more_ shuffled than normal.

Reply:
![image](https://user-images.githubusercontent.com/28734/225019544-8756b29a-632f-46ce-8af9-c9f41deb4716.png)